aml_service/pipelines/train_pipeline.py

```diff
@@ -1,164 +1,164 @@
-
-    print("The model path will be %s" % (mdl_path))
-
-    ws = get_workspace(
-        workspace_name,
-        resource_group,
-        subscription_id,
-        tenant_id,
-        app_id,
-        app_secret)
-    print(ws)
-
-    databricks_compute = get_compute(
-        ws,
-        databricks_compute_name,
-        resource_group,
-        databricks_workspace,
-        databricks_access_token)
-    print(databricks_compute)
-
-    step1 = DatabricksStep(
-        name="DBPythonInLocalMachine",
-        num_workers=1,
-        python_script_name=train_script_path,
-        source_directory=project_folder,
-        run_name='DB_Python_Local_demo',
-        existing_cluster_id=cluster_id,
-        compute_target=databricks_compute,
-        allow_reuse=False,
-        python_script_params=['--MODEL_PATH', mdl_path]
-    )
-
-    step2 = DatabricksStep(
-        name="RegisterModel",
-        num_workers=1,
-        python_script_name="Register.py",
-        source_directory=experiment_folder,
-        run_name='Register_model',
-        existing_cluster_id=cluster_id,
-        compute_target=databricks_compute,
-        allow_reuse=False,
-        python_script_params=[
-            '--MODEL_PATH', mdl_path,
-            '--TENANT_ID', tenant_id,
-            '--APP_ID', app_id,
-            '--APP_SECRET', app_secret,
-            '--MODEL_NAME', mdl_name]
-    )
-
-    step2.run_after(step1)
-    print("Step lists created")
-
-    pipeline = Pipeline(
-        workspace=ws,
-        # steps=[step1])
-        steps=[step1, step2])
-    print("Pipeline is built")
-
-    pipeline.validate()
-    print("Pipeline validation complete")
-
-    pipeline_run = pipeline.submit(experiment_name="pipetest")
-
-    print("Pipeline is submitted for execution")
-
-    pipeline_details = pipeline_run.get_details()
-
-    pipeline_run_id = pipeline_details['runId']
-
-    azure_run_url = get_experiment_run_url(
-        subscription_id,
-        resource_group,
-        workspace_name,
-        pipeline_run_id
-    )
-
-    print("To check details of the Pipeline run, go to " + azure_run_url)
-
-    pipeline_status = pipeline_run.get_status()
-
-    timer_mod = 0
-
-    while(pipeline_status == 'Running'):
-        timer_mod = timer_mod + 10
-        time.sleep(10)
-        if((timer_mod % 30) == 0):
-            print("Still running. %s seconds have passed." % (timer_mod))
-        pipeline_status = pipeline_run.get_status()
-
-    if pipeline_status == 'Failed':
-        sys.exit("AML Pipeline failed")
-    else:
-        print(pipeline_status)
-
-    print("Pipeline completed")
-
-
-if __name__ == '__main__':
-    main()
+from WorkSpace import get_workspace
+from AttachCompute import get_compute
+import sys
+import os
+import time
+import click
+import json
+
+from azureml.pipeline.core import Pipeline
+from azureml.pipeline.steps import DatabricksStep
+
+sys.path.append(os.path.abspath("./aml_service/experiment"))
```

Comment:
Hi @sptramer 

This refactoring was to comply with the rule [E042](https://lintlyci.github.io/Flake8Rules/rules/E402.html), I assume.

This structure unfortunately will break the code execution, as we first need to append the path to the folder `aml_service/experiment` before importing `WorkSpace` and `AttachCompute`.

Can you please update your changes to have this piece of code as follows:

```python
sys.path.append(os.path.abspath("./aml_service/experiment"))
from attach_compute import get_compute
from workspace import get_workspace
```